catalog/ui/catalog-ui-search/src/main/webapp/component/search-form/search-form.collection.view.js

```diff
@@ -0,0 +1,30 @@
+/**
+ * Copyright (c) Codice Foundation
+ *
+ * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser
+ * General Public License as published by the Free Software Foundation, either version 3 of the
+ * License, or any later version.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without
+ * even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details. A copy of the GNU Lesser General Public License
+ * is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ *
+ **/
+ /*global require*/
+ var Marionette = require('marionette');
+ var _ = require('underscore');
+ var $ = require('jquery');
+ var SearchFormView = require('./search-form.view');
+ var SearchFormCollection = require('././search-form-collection');
+ var CustomElements = require('js/CustomElements');
+
+ module.exports = Marionette.CollectionView.extend({
+     childView: SearchFormView,
+     tagName: CustomElements.register('my-search-forms'),
+     className: 'is-list is-inline has-list-highlighting',
+     initialize: function(options) {
```

Comment:
An easier way to pass these options on to the childView will be to use `childViewOptions`.  We do something similar in https://github.com/codice/ddf/blob/430880eb5d2c334ed74c78adb04d16e472225b82/catalog/ui/catalog-ui-search/src/main/webapp/component/result-item/result-item.collection.view.js#L37

Reply:
Done